src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -49,6 +50,28 @@ package object csv {
     }
   }
 
+  implicit class CsvRow(row: Row) {
```

Comment:
These methods would become exposed as part of our public interface. We don't want that, especially since they are fairly low-level.